packages/web/src/pages/_document.tsx

```diff
@@ -29,6 +29,12 @@ export default class Document extends NextDocument {
           <link rel="manifest" href="/manifest.json" />
           <meta name="msapplication-TileColor" content={colors.brandPrimary} />
           <meta name="msapplication-TileImage" content="/ms-icon-144x144.png" />
+          <link rel="preconnect" href="https://fonts.googleapis.com" />
```

Comment:
Can you try to follow [these docs](https://nextjs.org/docs/pages/building-your-application/optimizing/fonts) for importing fonts? I'm not sure it'll work but I don't think pulling from Google directly is the most efficient way to load fonts... if you have docs that say otherwise, please let me know!